Refit-Tests/InterfaceStubGenerator.cs

```diff
@@ -128,13 +128,13 @@ public interface IAmARefitInterfaceButNobodyUsesMe
         [Get("whatever")]
         Task RefitMethod();
 
-        [Refit.GetAttribute("something-else")]
+        [Get("something-else")]
```

Comment:
Formatting the document has broken this test case - it needs the `Refit.` because that's what's being tested.